config/core/build-configs.yaml

```diff
@@ -966,6 +969,20 @@ build_configs:
     tree: renesas
     branch: 'next'
 
+  riscv-fixes: &riscv-tree
```

Comment:
The convention is to use `tree_branch-name` so `riscv_fixes` and `riscv_for-next` would have been more correct.

Reply:
Fixed in PR #1577